profiles/seccomp/kernel_linux.go

```diff
@@ -0,0 +1,83 @@
+package seccomp
+
+import (
+	"bytes"
+	"fmt"
+	"sync"
+
+	"golang.org/x/sys/unix"
+)
+
+// kernelVersion holds information about the kernel.
+type kernelVersion struct {
+	kernel uint // Version of the kernel (i.e., the "4" in "4.1.2-generic")
+	major  uint // Major part of the kernel version (i.e., the "1" in "4.1.2-generic")
```

Comment:
This terminology seems unusual. Aren't they typically called "major" and "minor"?

Reply:
Yeah, the versioning is odd, but I think this answer gives a good summary https://askubuntu.com/a/843198, so the kernel is not using SemVer, but its own versioning scheme;

> #### Kernel Version . Major Revision . Minor Revision - Patch
>
> Using w.xx.y-zzz and looking at the fourth file listed /boot/vmlinuz-4.4.0-45 we can say:
>
> - w = Kernel Version = 4
> - xx= Major Revision = 4
> - y = Minor Revision = 0
> - zzz=Patch number = 45